semantic-python/src/Language/Python/Tags.hs

```diff
@@ -68,6 +68,9 @@ type family ToTagsInstance t :: Strategy where
   ToTagsInstance Py.NonlocalStatement  = 'Custom
   ToTagsInstance Py.PrintStatement     = 'Custom
 
+  -- Ignore for now to match a la carte tags output
+  ToTagsInstance Py.Interpolation      = 'Custom
```

Comment:
Precise ASTs do a much better job of handling syntax terms inside of things like interpolation, but for now I'm excluding tags here to match the current a la carte output. This would be a good thing to turn on later.

Reply:
👍 Good idea. https://github.com/github/semantic/issues/424